addons/payment_stripe/models/payment.py

```diff
@@ -149,14 +161,98 @@ def _get_feature_support(self):
         res['tokenize'].append('stripe')
         return res
 
+    def _handle_stripe_webhook(self, data):
+        """Process a webhook payload from Stripe.
+
+        Post-process a webhook payload to act upon the matching payment.transaction
+        record in Odoo.
+        """
+        SUPPORTED_WEBHOOKS = ['checkout.session.completed']
```

Comment:
That could be an interesting idea but I don't have specification for these events.
Also bear in mind that `payment_intent.succeeded` will occur right before `checkout.session.completed`, so we must pay attention not to duplicate any work or collide.

Reply:
since we will handle only this event then, the code can probably be simplified - let's not add some scaffolding if we don't actually need it, a list of one element is not useful :p

Reply:
**simplified code:**
```python
        wh_type = data.get('type')
        if wh_type != 'checkout.session.completed':
            _logger.info('unsupported webhook type %s, ignored', wh_type)
            return False

        _logger.info('handling %s webhook event from stripe', wh_type)
```